__snapshots__/contribution.js

```diff
@@ -0,0 +1,33 @@
+exports['contribution (contribution tests run for approximately 15 seconds) Tries to run contribution at reconcile with enough funds and met browsing requirements 1'] = [
+  {
```

Comment:
If I am not mistaken this is transaction object. Here should be the whole synopsis.

Reply:
Transactions are stored under `['ledger', 'info', 'transactions']`. Synopsis is stored at `['ledger', 'synopsis']. We don't get transactions in synopsis.

Reply:
We will return `['ledger', 'info']` for this.